docs/reproin.rst

```diff
@@ -0,0 +1,139 @@
+================
+Reproin 
+================
+
+This tutorial is based on `Dianne Patterson's University of Arizona tutorials <https://neuroimaging-core-docs.readthedocs.io/en/latest/pages/heudiconv.html#lesson-3-reproin-py>`_
+
+`Reproin <https://github.com/ReproNim/reproin>`_ is a setup for
+automatic generation of sharable, version-controlled BIDS datasets from
+MR scanners.
+
+If can control how your image sequences are named at the scanner you can
+use the *reproin* naming convention.  
+
+Get Example Dataset
+-------------------
+
+This example uses a phantom dataset: `reproin_dicom.zip <https://osf.io/4jwk5/>`_ generated by the University of Arizona on their Siemens Skyra 3T with Syngo MR VE11c software on 2018_02_08.
```

Comment:
should point to our copy at https://datasets.datalad.org/?dir=/repronim/heudiconv-reproin-example ?